inference-engine/src/transformations/include/transformations/snippets/scalar.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/constant.hpp"
+
+namespace ngraph {
+namespace op {
+
+/// \brief Class for constants.
+class TRANSFORMATIONS_API Scalar  : public Constant {
```

Comment:
Why do you need to have the new class for scalar constant?

Reply:
`scalar` is a constant which should be generated. For example constant in broadcast inputs are for shape and shouldn't  be present in generated assembly. Tensor Constants are passed as tensors, but scalar constants like scale/shift and power, are passed as Scalars.